### PR TITLE
docs: Update iOS install steps

### PR DESCRIPTION
**Pull request by jakubkoci**
Although I'm aware this is not the best way, It works for me. We can take it as a starting point and improve upon that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +127,21 @@ public class MainActivity extends ReactActivity {
 
 Add `Indy.framework` as dependency into your project. Beware that the library needs to be compiled with the same version of Swift as your project.
 
-React Native wrapper for iOS is written in Swift so you need to Create briding header. In Xcode app menu go to File -> New -> File... Select Swift File from dialog window and click Next and Finish. Xcode shoud ask you "Would you like to configure an Objective-C briding header?" then select Create Bridinging Header.
+Open `.xcworkspace` file in Xcode and in your project settings, tab General, section Frameworks, Libraries, and Embedded Content, click on plus. Then select Add Other -> Add files... and navigate to `Indy.framework` file on your disk.
+
+Create `Frameworks` folder in your project's `ios/Pods` directory and copy `Indy.frameworks` also there. (I think there should be a way how to avoid keeping `Indy.framework` in two places, but this way at least worked for me).
+
```
Comment (karimStekelenburg) [inline README.md:136]: Might be nice to add an additional step here stating that you need to run `pod install` in the `/ios` directory before moving on to the next step. If you don't, the `.xcconfig` files mentioned in the next step are non-existent (and the commands will fail) 😉.

File: README.md [hunk 2]
```diff
@@ -127,7 +127,21 @@ public class MainActivity extends ReactActivity {
 
 Add `Indy.framework` as dependency into your project. Beware that the library needs to be compiled with the same version of Swift as your project.
 
-React Native wrapper for iOS is written in Swift so you need to Create briding header. In Xcode app menu go to File -> New -> File... Select Swift File from dialog window and click Next and Finish. Xcode shoud ask you "Would you like to configure an Objective-C briding header?" then select Create Bridinging Header.
+Open `.xcworkspace` file in Xcode and in your project settings, tab General, section Frameworks, Libraries, and Embedded Content, click on plus. Then select Add Other -> Add files... and navigate to `Indy.framework` file on your disk.
+
+Create `Frameworks` folder in your project's `ios/Pods` directory and copy `Indy.frameworks` also there. (I think there should be a way how to avoid keeping `Indy.framework` in two places, but this way at least worked for me).
```
Comment (TimoGlastra) [inline README.md]: I copied `Indy.framework` to the `ios/Pods/Frameworks` directory, added that to `.gitignore.` and added that one in Xcode. This way you don't need the `Indy.framework` two times, but only one

File: README.md (hunk 1 above)
Comment (jakubkoci) [reply README.md:136]: Very good point. I added that as a first step 👍

File: README.md (hunk 2 above)
Comment (jakubkoci) [reply README.md]: Good tip. I haven't tried that, but I trust you 😄

Comment (TimoGlastra) [reply README.md]: Trust but verify 😉

https://github.com/animo/aries-mobile-agent-react-native/tree/develop/ios/Pods/Frameworks

File: README.md [hunk 3]
```diff
@@ -125,9 +125,26 @@ public class MainActivity extends ReactActivity {
 
 ## iOS
 
-Add `Indy.framework` as dependency into your project. Beware that the library needs to be compiled with the same version of Swift as your project.
+1. Install CocoaPods dependencies:
+```
+pod install --project-directory=ios/
+```
+
+2. Create `Frameworks` folder in your project's `ios/Pods` directory and copy `Indy.frameworks` into that directory.
```
Comment (TimoGlastra) [inline README.md:133]: `Indy.frameworks` -> `Indy.framework`

File: README.md [hunk 4]
```diff
@@ -125,9 +125,26 @@ public class MainActivity extends ReactActivity {
...
+
+3. Add `Indy.framework` as dependency into your project. Open `.xcworkspace` file in Xcode and in your project settings, tab General, section Frameworks, Libraries, and Embedded Content, click on plus. Then select Add Other -> Add files... and navigate to `Indy.framework` file on your disk. Beware that `Indy.framework` needs to be compiled with the same version of Swift as your project.
```
Comment (TimoGlastra) [inline README.md:135]: It is important to note that the file is not added to the gitignore. Otherwise you will need to do this after every clone

Comment (TimoGlastra) [reply README.md:135]: Also regarding the sentence:
"Beware that `Indy.framework` needs to be compiled with the same version of Swift as your project."

This is not true if "Indy.framework" is built with "Build Libraries for Distribution", which can be enabled by using:

`Select framework target -> Build Settings -> Build Libraries for Distribution (BUILD_LIBRARY_FOR_DISTRIBUTION) -> Yes`

See https://stackoverflow.com/a/63305234/10552895.

I think this is very useful to note

**Review by TimoGlastra — COMMENTED** [review]
Couple additional notes based on the merged version.

If you agree with the notes I can create an additional PR

File: README.md [hunk 5]
```diff
@@ -125,9 +125,26 @@ public class MainActivity extends ReactActivity {
...
+
+Debug build:
+```bash
+echo 'FRAMEWORK_SEARCH_PATHS = "${PODS_ROOT}/Frameworks"' >> ios/Pods/Target\ Support\ Files/rn-indy-sdk/rn-indy-sdk.debug.xcconfig
+```
+
+Release build:
+```bash
+echo 'FRAMEWORK_SEARCH_PATHS = "${PODS_ROOT}/Frameworks"' >> ios/Pods/Target\ Support\ Files/rn-indy-sdk/rn-indy-sdk.release.xcconfig
+```
```
Comment (karimStekelenburg) [inline README.md:147]: I've noticed that every time I add a React Native dependency that requires me to run `pod install`, these steps need to be repeated. I think the `pod install` command overwrites the `indy-sdk.*.xcconfig` files with the original files. Might be nice to add to the docs too :)

**Pull request merged**